apps/web/lib/org/[orgSlug]/instant-meeting/team/[slug]/[type]/getServerSideProps.ts

```diff
@@ -56,6 +56,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   return {
     props: {
       entity: eventData.entity,
+      eventTypeId: eventData.id,
```

Comment:
just pass the eventTypeId from getServersideProps to prevent waiting for it to load in public.event query and execute slots.getSchedule as soon as we can with the correct input